code/datums/mind.dm

```diff
@@ -829,30 +829,24 @@ datum/mind
 
 		else if (href_list["traitor"])
 			switch(href_list["traitor"])
-				if("clear")
+				if ("clear")
 					if(src in ticker.mode.traitors)
 						ticker.mode.traitors -= src
 						special_role = null
 						current << "\red <FONT size = 3><B>You have been brainwashed! You are no longer a traitor!</B></FONT>"
-						log_admin("[key_name_admin(usr)] has de-traitor'ed [current].")
+						log_admin("[key_name(usr)] has de-traitor'ed [key_name(current)].")

```

Comment:
key_name_admin is for href name that can be click (usually use for adminpms)
key_name is just a text (useful for log where no adminpms going to happen)